solutions/remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -393,6 +393,20 @@
                 "description": "Bing Maps region",
                 "comments": "Marked for deletion, unused"
             }
+        },
+        "deploymentId": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique Id of the deployment."
+            },
+            "defaultValue": "Undefined"
```

Comment:
what's the purpose of this default value?

Reply:
This is so that when we check-in to master and staging doesn't break. Right now it won't be sending any values for new parameters.